src/redux/utils/store.js

```diff
@@ -3,11 +3,11 @@ import thunk from "redux-thunk";
 import { apiMiddleware } from "redux-api-middleware";
 
 import reducers from "./reducers";
-import DevTools from "./devtools";
+//import DevTools from "./devtools";
 
 const enhancer = compose(
-  applyMiddleware(thunk, apiMiddleware),
-  DevTools.instrument()
+  applyMiddleware(thunk, apiMiddleware)
+  //DevTools.instrument()
```

Comment:
I am going to keep them in case I need to re-enable during development and use them. The webpack config removes comments in production so we won't really have to worry.